core/lib/db_connection/src/connection_pool.rs

```diff
@@ -24,25 +25,14 @@ use crate::{
 };
 
 /// Builder for [`ConnectionPool`]s.
-#[derive(Clone)]
+#[derive(derive_more::Debug, Clone)]
```

Comment:
Not sure whether using `derive_more` makes sense here; it'd be relatively easy to implement `Debug` manually (although in this case it's easy for this impl to become stale).